File: PHP/1768. Merge Strings Alternately.php
<?php
// 1768. Merge Strings Alternately
// Level: Easy
// https://leetcode.com/problems/merge-strings-alternately/

class Solution
{
  function mergeAlternately($word1, $word2)
  {
    $str = "";
    for ($i = 0; $i < strlen($word1) || $i < strlen($word2); $i++) {
      if ($i < strlen($word1)) {
        $str .= $word1[$i];
      }
      if ($i < strlen($word2)) {
        $str .= $word2[$i];
      }
    }
    return $str;
  }
}

File: CSharp/1768. Merge Strings Alternately.cs
// 1768. Merge Strings Alternately
// Level: Easy
// https://leetcode.com/problems/merge-strings-alternately/

public class Solution
{
  public string MergeAlternately(string word1, string word2)
  {
    StringBuilder str = new StringBuilder();
    for (int i = 0; i < word1.Length || i < word2.Length; i++)
    {
      if (word1.Length > i)
        str.Append(word1[i]);
      if (word2.Length > i)
        str.Append(word2[i]);
    }
    return str.ToString();
  }
}